docs/examples/notebooks/03-Integrator.ipynb

```diff
@@ -0,0 +1,214 @@
+    "    fpga_ens = FpgaPesEnsembleNetwork(\n",
+    "        board,  # The board to use (from above)\n",
+    "        n_neurons=100,  # The number of neurons to use in the ensemble\n",
+    "        dimensions=1,  # 1 dimensions\n",
+    "        learning_rate=0,  # No learning for this example\n",
+    "        feedback=1  # Activate the recurrent connection\n",
+    "    )\n",
+    "\n",
+    "    # Setting `feedback=1` creates a `feedback` connection object\n",
+    "    # with the identity transform. You can set the attributes on this\n",
+    "    # feedback connection object, such as `.function` and `.transform`\n",
+    "    fpga_ens.feedback.synapse = tau  # `nengo.LowPass(tau)`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Connect Everything Together\n",
+    "The recurrent connection is housed on the FPGA device and is created as part of the `FpgaPesEnsembleNetwork` object, so the only connection that needs to be made is the input stimulus to the FPGA ensemble."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    # Connect the stimulus\n",
+    "    nengo.Connection(input_node, fpga_ens.input,\n",
+    "                     synapse=tau, transform=tau)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 6: Add Probes to Collect Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    # The original input\n",
+    "    input_p = nengo.Probe(input_node)\n",
+    "\n",
+    "    # The output from the FPGA ensemble\n",
+    "    # (filtered with a 10ms post-synaptic filter)\n",
+    "    output_p = nengo.Probe(fpga_ens.output, synapse=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7: Run the Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with nengo_fpga.Simulator(model) as sim:\n",
+    "    sim.run(6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 8: Plot the Results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8, 5))\n",
+    "plt.plot(sim.trange(), sim.data[input_p], 'k--', label=\"Input\")\n",
+    "plt.plot(sim.trange(), sim.data[output_p], label=\"Integrator output\")\n",
+    "plt.legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The plot above shows the neurons effectively integrating the input signal, albeit with a little drift. Running several times will give a sense of the kinds of drift you might expect. Drift can be reduced by increasing the number of neurons.\n",
```

Comment:
The plot above shows the neurons effectively integrating the input signal. Because of the implementation in neurons, the integration is not perfect (i.e., there will be some drift). Run the simulation several times to get a sense of the kinds of drift you might expect. 

Try playing around with the number of neurons in the FPGA ensemble as well as the synaptic time constant (`tau`) to see how it effects performance (e.g., observe how changing these numbers affect the drift)!